src/mlte/spec/condition.py

```diff
@@ -0,0 +1,108 @@
+"""
+Defines a condition for a measurement to be approved, including the validator to use and the needed threshold.
+"""
+
+from __future__ import annotations
+
+from typing import Any
+
+from mlte.validation import Result
+from mlte.value import Value
+
+# -----------------------------------------------------------------------------
+# Condition
+# -----------------------------------------------------------------------------
+
+
+class Condition:
+    """
+    The Condition class defines a relation between a measurement,
+    a validation method and a threshold/parameter, that will be part of a spec.
+    """
+
+    def __init__(
+        self,
+        label: str,
+        measurement_type: str,
+        validator: str,
+        threshold: Any,
```

Comment:
Do we need to think of something more "rigorous" for specifying thresholds?

Reply:
Definitely, and that will come with the next PR and thunks and everything as we had discussed.